packages/e2e/src/specs/messages/messages-search.e2e.ts

```diff
@@ -0,0 +1,407 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import navigationTabs from "../../page-objects/tabs.page"
+import messagesPage from "../../page-objects/messages.page"
+import modalMessagesPage from "../../page-objects/modal-messages.page"
+import modalGeneralPage from "../../page-objects/modal-general.page"
+import messagesConversationPage from "../../page-objects/messages-conversation.page"
+
+import {
+  sendMessage,
+  deleteAllMessagesWithSelectionManager,
+} from "../../helpers/messages.helper"
+import {
+  addNewContact,
+  deleteContactsWithSelectionManager,
+} from "../../helpers/contacts.helper"
+import { waitForClickableAndClick } from "../../helpers/general.helper"
+
+const enterKey = "\uE007"
+const contacts: Array<{
```

Comment:
I think test data can be kept in a separate file

Reply:
https://github.com/mudita/mudita-center/pull/1172/commits/8fbeb0f32444f86724c5a886e8deea4622e18f30